assemblies/assembly-configuring-authorization-in-rhdh.adoc

```diff
@@ -11,25 +11,25 @@ Role-Based Access Control (RBAC) is a security concept that controls access to r
 You define roles with specific permissions, and then assign the roles to users and groups.
 
 RBAC on {product-short} is built on top of the Permissions framework, which defines RBAC policies in code.
-Rather than defining policies in code,
-the {product-short} RBAC feature allows you
-to define policies in a declarative fashion using a simple CSV based format.
+Rather than defining policies in code, the {product-short} RBAC feature allows you to define policies in a declarative fashion using a simple CSV based format.
```

Comment:
```suggestion
Rather than defining policies in code,
the {product-short} RBAC feature allows you
to define policies in a declarative fashion using a simple CSV based format.
```